### ISSUE TITLE
Copy editor can not view assigned submission for copy editing.

### ISSUE THREAD

**Issue by touhidurabir** [pkp/pkp-lib#7907]
**Describe the bug**
After the submission assigned to a `copy editor` user, if they copy editor user try to view it , a blank page is showing which points out an internal exception .

**To Reproduce**
1. Create a new Submission.
2. Complete up to the review process .
3. Create a new user with `copy editor` role .
4. Assign the copy editor user to the submission .
5. Login as the copy editor and try to view the assigned submission which will present a blank page .

**What application are you using?**
 OJS `3.3.0` . This issue confirmed in both `stable-3_3_0` and `main` branch . 

**Additional information**
The issue originated from `pages/workflow/PKPWorkflowHandler.inc.php` with internal exception as `array_intersect(): Argument #2 must be of type array, null given` . This issue only occur with running `PHP 8.0+` version for the installation .

**Comment by touhidurabir** [comment]
@asmecher Here is the PRs list . Only did it for `pkp-lib` and `ojs` of both `stable-3_3_0` and `main` branch . Once reviewed and confirmed , I will forward the changes for 'ops' and `omp` of both `stable-3_3_0` and `main` branches.

### `stable-3_3_0` PRs

[pkp-lib](https://github.com/pkp/pkp-lib) --> [pkp/pkp-lib#7907 Copy editor can not view assigned submission for copy editing](https://github.com/pkp/pkp-lib/pull/7910)
[ojs](https://github.com/pkp/ojs) --> [pkp/pkp-lib#7907 Submodule Update ##touhidurabir/i7907_fix_stable_3_3_0##](https://github.com/pkp/ojs/pull/3391)

### `main` PRs

[pkp-lib](https://github.com/pkp/pkp-lib) --> [pkp/pkp-lib#7907 Copy editor can not view assigned submission for copy editing](https://github.com/pkp/pkp-lib/pull/7911)
[ojs](https://github.com/pkp/ojs) --> [pkp/pkp-lib#7907 Submodule Update ##touhidurabir/i7907_fix_stable_3_3_0##](https://github.com/pkp/ojs/pull/3392)

**Comment by NateWr** [comment]
Thanks @touhidurabir! I think this is a duplicate of https://github.com/pkp/pkp-lib/issues/7822. I'll let Alec sort out what to merge.

**Comment by touhidurabir** [comment]
@NateWr you are right and the PR that @asmecher has made for the issue https://github.com/pkp/pkp-lib/issues/7822 has more broad impact as that solve all such cases for exception `array_intersect(): Argument #2 must be of type array, null given` . 

@asmecher if you confirm, I will close this issue and PRs related to this one .
